src/renderers/shared/fiber/ReactReifiedYield.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactReifiedYield
+ * @flow
+ */
+
+'use strict';
+
+import type { ReactYield } from 'ReactCoroutine';
+import type { Fiber } from 'ReactFiber';
+
+var ReactFiber = require('ReactFiber');
+
+type ReifiedYield = { continuation: Object, props: Object };
```

Comment:
Should this have a $$typeof? It gets exposed to user code in the coroutine handler.